digdag-standards/src/main/java/io/digdag/standards/command/EcsCommandExecutor.java

```diff
@@ -439,7 +439,22 @@ CommandStatus createNextCommandStatus(
             // finish this poll once and wait finish marker in head of this method in next poll, considering risk of crushing in this poll.
             nextStatus.put("task_finished_at", Instant.now().getEpochSecond());
             // Set exit code of container finished to nextStatus
-            nextStatus.put("status_code", task.getContainers().get(0).getExitCode());
+            int containerExitCode = task.getContainers().get(0).getExitCode().intValue();
+            nextStatus.put("status_code", containerExitCode);
+
+            String stopCode = task.getStopCode();
+            nextStatus.put("ecs_stop_code", stopCode);
+
+            // see com.amazonaws.services.ecs.model.TaskStopCode
+            if(TaskStopCode.TaskFailedToStart.toString().equals(stopCode)) {
+                String stoppedReason = task.getStoppedReason();
+                nextStatus.put("ecs_stopped_reason", stoppedReason);
+                throw new RuntimeException(s(
```

Comment:
If container failed to launch, no LogStream is created. So, avoid retrying to fetch Cloudwatch logs by throwing fail-fast error.